packages/core/hooks/ac_parser.py

```
@@ -0,0 +1,99 @@
+
+from __future__ import annotations
+
+import re
+
+__all__ = [
+    "CHECKBOX_PATTERN",
+    "AC_SECTION_HEADING",
+    "classify_checkbox_line",
+    "ac_section_ranges",
+    "phase_has_unchecked_ac",
+]
```

Ruff (0.15.21)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)